app/com/linkedin/drelephant/tuning/AutoTuningAPIHelper.java

```diff
@@ -137,32 +168,111 @@ public void setDefaultValue(TuningInput tuningInput) {
     tuningInput.setTuningAlgorithm(tuningAlgorithm);
   }
 
+  /**
+   * Applies penalty to the given execution
+   * @param jobExecId String jobExecId of the execution to which penalty has to be applied
+   */
+  private void applyPenalty(String jobExecId) {
+    Integer penaltyConstant = 3;
```

Comment:
Add a comment in the code on why 3 was chosen?

Reply:
Added.